ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -3016,10 +3080,12 @@ let getHeadmarkerId = (data, matches) => {
       'locale': 'fr',
       'missingTranslations': true,
```

Comment:
Are there still missing translations here?

Reply:
no, this is a hesitation. There is nothing left to translate. But there is this little regex problem in `effectname`  which I told you about, or you would need the id for netregex (gains and lose effect for TEA Decree Nisi), i think it's only in fr this prob.
I have removed this line.

Reply:
I will take care of this.  I took care of arr/hw in https://github.com/quisquous/cactbot/commit/8b560cff98da4cfc909c1641be0110d0f0cf1145 and I will try to get to the rest this weekend.

Reply:
Wow, that is a lot of work. But it is a good improvement which will avoid many worries ^^. Thank you.